modules/renter/hostdb/hostentry.go

```diff
@@ -120,7 +120,7 @@ func (hdb *HostDB) IsOffline(addr modules.NetAddress) bool {
 		return false
 	}
 	if h, ok := hdb.allHosts[addr]; ok {
-		return !h.online
+		return !h.Online
 	}
 	// no record of the host; add it to the HostDB
 	hdb.insertHost(modules.HostSettings{NetAddress: addr})
```

Comment:
might not be a good choice, especially if the user is trying to keep a filter on which hosts are getting into the hostdb. On the other hand, insertHost might be the right place to add the filtering code, so maybe this is okay.

Should insertHost be called in a goroutine, I don't think  you want to block while it does whatever blackbox actions.